modules/system/defaults/NSGlobalDomain.nix

```diff
@@ -7,6 +7,21 @@ let
   inherit (config.lib.defaults.types) floatWithDeprecationError;
 in {
   options = {
+    system.defaults.NSGlobalDomain.AppleActionOnDoubleClick = mkOption {
+      type = types.nullOr (
+        types.enum [
+          "Minimize"
+          "Zoom"
+          "Fill"
+          "None"
+        ]
+      );
+      default = null;
+      description = ''
+        Behaviour of double clicking a windows title bar.
```

Comment:
Please add a note in here that the System Settings option "Zoom" corresponds to the defaults value "Maximize".

also, nit: "windows" should be "window's"